src/lib.rs

```diff
@@ -428,6 +428,24 @@ impl Build {
         self
     }
 
+    /// removes a flag from the existing flags.
+    /// example:
+    /// ```no_run
+    /// cc::Build::new()
+    ///     .file("src/foo.c")
+    ///     .flag("-fake_flag")
+    ///     .remove_flag("-fake_flag")
+    ///     .compile("foo");
+    /// ```
+    pub fn remove_flag(&mut self, flag: &'static str) -> &mut Self{
```

Comment:
The `flag` doesn't have to be a `'static`

```suggestion
    pub fn remove_flag(&mut self, flag: &str) -> &mut Self{
```